lib/segment/src/index/plain_payload_index.rs

```diff
@@ -232,18 +232,23 @@ impl PlainIndex {
         filter: Option<&Filter>,
     ) -> bool {
         let vector_storage = self.vector_storage.borrow();
-        let vector_size_bytes = vector_storage.vector_dim() * VECTOR_ELEMENT_SIZE;
-        let indexing_threshold_bytes = search_optimized_threshold_kb * BYTES_IN_KB;
-
-        if let Some(payload_filter) = filter {
-            let payload_index = self.payload_index.borrow();
-            let cardinality = payload_index.estimate_cardinality(payload_filter);
-            let scan_size = vector_size_bytes.saturating_mul(cardinality.max);
-            scan_size <= indexing_threshold_bytes
+        let available_vector_count = vector_storage.available_vector_count();
```

Comment:
Here `vector_size_bytes` was changed only. If storage is empty, the function returns `true` - empty storage is small enough for unindexed search